apt/method.go

```diff
@@ -66,28 +68,33 @@ type Method struct {
 
 type aptMethodConfig struct {
 	serviceAccountJSON, serviceAccountEmail string
+	debug                                   bool
 }
 
 // Run runs the method.
-func (m *Method) Run(ctx context.Context) {
+func (m *Method) Run(ctx context.Context) error {
 	m.writer.SendCapabilities()
 	for {
 		select {
 		case <-ctx.Done():
-			return
+			return nil
 		default:
 		}
 		msg, err := m.reader.ReadMessage(ctx)
-		if err != nil {
+		if err == errEmptyMessage {
```

Comment:
please `errors.Is()` instead.

Reply:
Oh interesting, a new feature added in go1.13. Sure, I'll use it. Done.